src/contracts/minter/facets/LiquidationFacet.sol

```diff
@@ -72,19 +72,20 @@ contract LiquidationFacet is DiamondModifiers, ILiquidation {
                 _repayKreskoAsset,
                 _collateralAssetToSeize
             );
-
             require(krAssetDebt >= _repayAmount, Error.KRASSET_BURN_AMOUNT_OVERFLOW);
             require(repayAmountUSD.isLessThanOrEqual(maxLiquidation), Error.LIQUIDATION_OVERFLOW);
         }
 
         FixedPoint.Unsigned memory collateralPriceUSD = FixedPoint.Unsigned(
             uint256(s.collateralAssets[_collateralAssetToSeize].oracle.latestAnswer())
         );
+        // Charge burn fee from the liquidated user
```

Comment:
```suggestion
        // Charge close fee from the user targeted for liquidation
```

Reply:
nit, but user hasn't been liquidated yet since we're assessing the fee before the actual liquidation